e2/command/status.go

```diff
@@ -0,0 +1,27 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/suborbital/e2core/e2/util"
+)
+
+func StatusCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "status",
+		Short: "Get K8s deployment status",
+		Long:  "Get Kubernetes pods and service status for your E2 Core deployment",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			util.LogInfo("Pods:")
+			util.Command.Run("kubectl get pods -n suborbital")
+
+			fmt.Println()
+			util.LogInfo("Services:")
+			util.Command.Run("kubectl get svc -n suborbital")
+
+			return nil
```

Comment:
Let's also try starting up the kubectl proxy and pinging the e2core health endpoint to make sure it's actually started and serving?

Reply:
I think we'd just pull metadata from Kubernetes itself to ensure the services are healthy

Reply:
I'll make some changes to the templates to support this.

Reply:
https://github.com/suborbital/templates/pull/20